examples/junit5-arquillian-simple-websockets/src/test/java/resources/arquillian.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="
+              http://jboss.org/schema/arquillian
+              http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+    <container qualifier="tomee" default="true">
+        <configuration>
+            <property name="serverXml">src/main/conf/server.xml</property>
+            <property name="httpPort">8081</property>
```

Comment:
I recall fixing port numbers can lead to concurrency issues during test executions. 
The port definition in arquillian.xml can be dynamic. 

Example
```      <property name="httpPort">-1</property>
      <property name="stopPort">-1</property>
```
https://github.com/apache/tomee/blob/master/examples/mp-custom-healthcheck/src/test/resources/arquillian.xml#L22-L23

Reply:
You are right. If CI executes multiple jobs in parallel, it might clash with ports. We normally work with Gitlab CI and isolated containers. Nice catch! I fixed this.